programs/governance-rewards/src/instructions/reclaim/reclaim_user_data.rs

```diff
@@ -0,0 +1,34 @@
+use anchor_lang::prelude::*;
+
+use crate::{
+    error::GovernanceRewardsError,
+    state::{claim_data::ClaimData, distribution::Distribution},
+};
+
+#[derive(Accounts)]
+pub struct ReclaimUserData<'info> {
+    distribution: Account<'info, Distribution>,
+
+    #[account(
+        mut,
+        close = caller,
+    )]
+    claim_data: Account<'info, ClaimData>,
+
+    /// CHECK: Not read
+    caller: AccountInfo<'info>,
```

Comment:
if this is free-for-all, won't everyone race to close accounts? (sorry if you've already replied to this in the old PR, if you did I didn't see it)

Reply:
Yeah I will change this so the funds go back to the creating user.